person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :age, :name
+  attr_reader :id
+
+  def can_use_services?
+    return true if of_age? || Person.parent_permission
+
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if Person.age < 18
```

Comment:
- [ ] I get undefined method error for `age` here. Kindly use the `@age` instance variable instead

- [ ] Right now, your method returns `true` when the age is less than 18. Whereas, the method should return `true` when the age is **greater than or equal to** 18. Kindly fix that

  **SUGGESTION:** You should have something like this:
  ```
  def of_age?
    return true if @age >= 18
    ...
  ```